data/data/openstack/service/main.tf

```diff
@@ -168,20 +168,20 @@ data "ignition_file" "coredb" {
   content {
     content = <<EOF
 $ORIGIN ${var.cluster_domain}.
-@    3600 IN SOA host-${var.cluster_name}.${var.cluster_domain}. hostmaster (
+@    3600 IN SOA host.${var.cluster_domain}. hostmaster (
                                 2017042752 ; serial
                                 7200       ; refresh (2 hours)
                                 3600       ; retry (1 hour)
                                 1209600    ; expire (2 weeks)
                                 3600       ; minimum (1 hour)
                                 )
 
-${var.cluster_name}-api  IN  A  ${var.service_vm_floating_ip}
-*.apps.${var.cluster_name}  IN  A  ${var.service_vm_floating_ip}
+api  IN  A  ${var.service_vm_floating_ip}
+*.apps  IN  A  ${var.service_vm_floating_ip}
```

Comment:
`var.service_vm_floating_ip` is now `var.lb_floating_ip`

Reply:
Thanks! done with https://github.com/openshift/installer/pull/1169/commits/15dfaa2b7c7422b2c7e686b6bdce5895ccc9beb2 -> https://github.com/openshift/installer/pull/1169/commits/6a84e7bc0575f2009765d49229c771e471a9f09f